_schema.graphql

```diff
@@ -2598,23 +2598,6 @@ type dimensions {
   cm: String
 }
 
-# Partner
-type DoNotUseThisPartner {
```

Comment:
From names it seems like these are safe to remove, but it's not clear to me what triggered their removal?

Reply:
I just ran into exact same issue, you need to rebase with latest `master` and then do a `yarn dump-schema _schema.graphql` that should fix this.

Reply:
I was following that conversation, and I am up to date against the latest master. These aren't Kaws types getting removed as far as I can tell, though I'm unsure why your PR didn't also remove these.